packages/phoenix-evals/src/phoenix/evals/span_templates.py

```diff
@@ -186,6 +186,86 @@
 
 QA_SPAN_PROMPT_RAILS_MAP = OrderedDict({True: "correct", False: "incorrect"})
 
+FUNCTION_CALLING_SPAN_BASE_TEMPLATE = """
+    You are an evaluation assistant evaluating questions and function calls to
+    determine whether the function called would answer the question. The function
+    calls have been generated by a separate agent, and chosen from the list of
+    functions provided below. It is your job to decide whether that agent chose
+    the right function to call. The agent can also choose not to call a function,
+    which it should do if there is no appropriate function provided, or if the
+    question is missing key parameters needed to call that function.
```

Comment:
This won't always be true. You can force the API to make a function/ tool call. https://platform.openai.com/docs/api-reference/chat/create#chat-create-tool_choice